scripts/ci.sh

```diff
@@ -12,23 +12,23 @@ tilt ci
 
 if [ "${KKA_DEPLOY_MINIMAL}" == "false" ]; then
   # Login on ArgoCD
-  argocd login --core --name k8s-kurated-addons
   kubectl config set-context --current --namespace=argocd
+  argocd login --core --name k8s-kurated-addons
 
   # Ensure ArgoCD apps are all healthy and in sync
   echo ">> Waiting for k8s-kurated-addons to be healty and in sync..."
   while [ true ]
   do
     ALL_HEALTHY=true
-    readarray -t apps_health < <(argocd app get k8s-kurated-addons -o json | jq -r '.status.resources | .[]? | select(.kind | contains("Application")) | .health.status')
+    readarray -t apps_health < <(argocd app get k8s-kurated-addons -o json | jq -r '.status.resources | .[]? | select(.kind | contains("Application")) | .health')
```

Comment:
did the argocd cli change the response structure?

Reply:
rolled back this